client/src/components/Tooltip/Tooltip.scss

```diff
@@ -11,3 +11,77 @@
   }
 }
 
+/**
+ * Tooltip arrow style below is copied (unless stated otherwise) from Bootstrap
+ * v4 beta because there's a missmatch between classname in react-bootstrap and
+ * its counterpart in Bootstrap v4.
```

Comment:
could this use `@extend` so it's not hardcoded-copying? use `// sass-lint:disable-block extend-rule-name` for linting with the comment above

Reply:
This requires us to extends `.tooltip .arrow` but unfortunately we can't extend a nested selector.